packages/common/src/projects/_internal/ProjectBusinessRules.ts

```diff
@@ -35,6 +36,11 @@ export const ProjectCapabilityPermissions: ICapabilityPermission[] = [
     capability: "settings",
     permissions: ["hub:project:owner"],
   },
+  {
+    entity: "project",
+    capability: "metrics",
+    permissions: ["hub:project:owner"],
```

Comment:
I think we want to limit the project metrics capability to `"hub:project:edit"` rather than `"hub:project:owner"`